src/Compilers/VisualBasic/Portable/CodeGen/CodeGenerator.vb

```diff
@@ -261,7 +287,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.CodeGen
                 instructionsEmitted = EmitStatementAndCountInstructions(statement)
             End If
 
-            If instructionsEmitted = 0 AndAlso span <> Nothing AndAlso _optimizations = OptimizationLevel.Debug Then
+            If instructionsEmitted = 0 AndAlso span <> Nothing AndAlso _ilEmitStyle = OptimizationLevel.Debug Then
```

Comment:
Yes, this is not a type. There are statements that could result in no code emitted. For example labels. We may also have a case of sequence point not wrapping a statement at all - like when it is associated with "End If"